Dockerfile

```diff
@@ -97,13 +95,17 @@ RUN yum update -y && \
     git config --global user.email "tfmod442916@users.noreply.github.com" && \
     git config --global user.name "github-actions[bot]" && \
     git config --global --add safe.directory '*'
-RUN mkdir /home/runtimeuser && \
-    chmod -R 777 /home/runtimeuser
-COPY .terraformrc /home/runtimeuser/.terraformrc
+# Create home directory, copy over utilities for xyzenv, terraform cli config, and set permissions
+RUN mkdir ${HOME_DIR}
+COPY .terraformrc ${HOME_DIR}/.terraformrc
 COPY --from=build /go/bin /usr/local/go/bin
-COPY --from=build /src/tfenv /home/runtimeuser/tfenv
-RUN chmod 777 /home/runtimeuser/tfenv && \
-    git clone https://github.com/iamhsa/pkenv.git /pkenv && \
-    cd /pkenv && rm -rf .git && \
+COPY --from=build /src/tfenv ${HOME_DIR}/tfenv
+COPY --from=build /src/pkenv ${HOME_DIR}/pkenv
+RUN mkdir ${HOME_DIR}/tflintenv && \
+    mkdir -p ${HOME_DIR}/.terraform.d/plugin-cache && \
+    chmod -Rv a+rwX ${HOME_DIR} && \
+    chmod 777 ${HOME_DIR}/tfenv/bin/* && \
+    chmod 777 ${HOME_DIR}/pkenv/bin/* && \
     rm -r /tmp/* && \
     yum clean all
+ENV HOME=${HOME_DIR}
```

Comment:
What is this used for?

Reply:
Sets home dir on container to /home/runtimeuser as opposed to your actual home directory from the passwd file, which won't exist